src/views/Point.js

```diff
@@ -140,95 +228,54 @@ export default function Point() {
         animationMode={'slide'}
       />
       <Grid gap={3}>
-        {isPlanet && hasInvites && (
-          <>
-            <Grid.Item cols={[1, 11]}>
-              Invite Group
-              <br />
-            </Grid.Item>
-
-            <Grid.Item
-              className={cn('t-right underline pointer-hover', {
-                gray4: sentInvites.getOrElse(0) === 0,
-              })}
-              onClick={goCohort}
-              cols={[11, 13]}>
-              View
-            </Grid.Item>
-            <Grid.Item full>
-              <Flex align="center">
-                <Flex.Item>
-                  {acceptedInvites.getOrElse(0)} / {_totalInvites}
-                </Flex.Item>
-                {_pendingInvites > 0 && (
-                  <Flex.Item as={Chip} className="bg-yellow1 yellow4">
-                    {_pendingInvites} pending
-                  </Flex.Item>
-                )}
-              </Flex>
-            </Grid.Item>
-
-            {showInvites && (
-              <>
-                <Grid.Item
-                  full
-                  as={BarGraph}
-                  available={availableInvites}
-                  sent={sentInvites}
-                  accepted={acceptedInvites}
-                />
-                <Grid.Item
-                  full
-                  as={InviteSigilList}
-                  pendingPoints={pendingPoints}
-                  acceptedPoints={acceptedPoints}
-                />
-              </>
-            )}
-            {!showInvites && (
-              <>
-                <Grid.Item full className="b-gray4 b-dotted b1 self-center">
-                  <div className="p4 pv8 t-center gray4">
-                    Start your invite group by adding members
-                  </div>
-                </Grid.Item>
-              </>
-            )}
-            {!showInviteForm && availableInvites.getOrElse(0) > 0 && (
-              <Grid.Item
-                full
-                solid
-                as={Button}
-                center
-                onClick={() => setShowInviteForm(true)}>
-                Add Members
-              </Grid.Item>
+        {isParent && (
+          <Grid.Item
+            full
+            as={Flex}
+            justify="between"
+            onClick={goResidents}
```

Comment:
This should get a `class="pointer"` or something, it doesn't currently signal clickability in any way. May also want an arrow at the end of the item?